test/tests/postprocessors/element_integral_function/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  [./test]
+    type = 'CSVDiff'
+    input = 'element_integral_function.i'
+    csvdiff = 'element_integral_function_out.csv'
+    abs_zero = 1e-13
+
+    requirement = "ElementIntegralFunction shall correctly integrate functions."
```

Comment:
Seeing the future. Design comes _after_ requirements. Also correctness is business for verification not for requirements definition. 
The requirement here is "The system shall integrate functions." You could say "volumetric functions", which would be more precise. I can see something like `FunctionSideIntegral`, which - as I just found out - we already have. In that case I would also suggest to change the name of the PPS to `FunctionElementIntegral`. The 'Function' suffix is reserved for objects that inherit from `Function`.